addons/survey/report/survey_templates.xml

```diff
@@ -37,7 +37,7 @@
                                 </t>
                                 <span t-att-style="certif_style" class="user-name" t-out="certified_name">DEMO_CERTIFIED_NAME</span>
 
-                                <br/> <span>by</span> <span class="certification-company" t-field="user_input.create_uid.company_id.display_name">Odoo</span> <span>for successfully completing</span>
+                                <br/> <span>by</span> <span class="certification-company" t-field="user_input.create_uid.company_id.sudo().display_name">Odoo</span> <span>for successfully completing</span>
```

Comment:
It's preferred to use the sudo on `create_uid` since having the Public user set to another company where we don't have right we don't have access to that `res.user`
```suggestion
                                <br/> <span>by</span> <span class="certification-company" t-field="user_input.create_uid.sudo().company_id.display_name">Odoo</span> <span>for successfully completing</span>
```